aiosu/utils/performance.py

```diff
@@ -93,13 +94,42 @@ def calculate(self, score: Score) -> OsuPerformanceAttributes:
         if score.beatmap is None:
             raise ValueError("Given score does not have a beatmap.")
 
-        effective_miss_count = self._calculate_effective_miss_count(score)
         total_hits = (
             score.statistics.count_300
             + score.statistics.count_100
             + score.statistics.count_50
             + score.statistics.count_miss
         )
+        effective_miss_count = score.statistics.count_miss
+
+        if score.beatmap.count_sliders > 0:  # type: ignore
+            if self._is_slider_head_accuracy(score):
+                full_combo_threshold = self.difficulty_attributes.max_combo - 0.1 * score.beatmap.count_sliders  # type: ignore
+
+                if score.max_combo < full_combo_threshold:
+                    effective_miss_count = full_combo_threshold / max(
```

Comment:
```suggestion
                    effective_miss_count = full_combo_threshold // max(
```

not sure if this is right though?

Reply:
effective misscount is a decimal number